enclave_wrangler/objects_api.py

```diff
@@ -353,8 +356,17 @@ def fetch_cset_and_member_objects(
     for cset in cset_versions:
         version: int = cset['properties']['codesetId']
         # Hierarchical
-        cset['expression_items']: List[Dict] = get_concept_set_version_expression_items(version, return_detail='full')
-        cset['member_items']: List[Dict] = get_concept_set_version_members(version, return_detail='full')
+        # TODO: exc handling: Will need to handle differently or do more: https://github.com/jhu-bids/TermHub/issues/451
+        try:
+            cset['expression_items']: List[Dict] = get_concept_set_version_expression_items(version, return_detail='full')
+        except EnclavePaginationLimitErr as err:
```

Comment:
Can Amin and co really not fix the `ObjectsExceededLimit` error? I can't remember why. Was this a Foundry thing?

Reply:
Btw, this limit happened only when fetching expression items, and not during member items. I wonder if this is a coincidence, or this is just how it is w/ Enclave; that the expression items has a limit of 10k items returned, and member items do not have such a limit.

Currently I'm debugging codeset `938394329`, and so far it has 51,000 unique items. I think this may be what you were stuck on when you thought you were in an endless loop fetching the same data over and over; I don't think that's the case right now. I assume that this will eventually complete.